### ISSUE TITLE
(Classic Raid) Molten Core Bugs-Vanilla (Guild)

### ISSUE THREAD

**Issue by chaostheory102** [dalaranwow/dalaran-wow#4111]
[Molten Core Bug1.docx](https://github.com/dalaranwow/dalaran-wow/files/683906/Molten.Core.Bug1.docx)
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Ty very much! iI saved this file to my ToDo-List. I will open individual tickets for each issue and push the fix for it :)

**Comment by chaostheory102** [comment]
Sorry bout this but i forgot 1 main bug!
Garr
       Current Behavior
       Firesworn do not buff Garr on death
       Expected Behavior
       Firesworn should buff Garr on death, causing a crowd control
       Steps to Reproduce
       1. Kill adds
       2. View as buff does not get placed on Garr

**Comment by Rushor** [comment]
kk added it to the list - ty!

**Comment by chaostheory102** [comment]
Thank you!
I am working on the bug list for the other vanilla raids, should i continue
to post on the bugtracker or email the finished lists to you?

**Comment by Rushor** [comment]
bugtracker is fine

**Comment by Rushor** [comment]
i will copy the whole issue here, to make indivisual issues tomorrow:

Molten Core Bugs
Core Hound Packs – 4 problems
	Problem 1
		Current Behavior
Core Hounds in Magmadar’s Cavern do not resurrect one another when they are not killed within 10 second intervals
Expected Behavior
Core Hounds should revive one another if they 10 seconds have passed after one of them have died and 1 other member of the pack is still alive
Steps to Reproduce Problem
	1.Kill one of the core hound in the pack
	2. Wait 10 seconds
	3. Observe that it is not being respawned
		Proof
	http://wowwiki.wikia.com/wiki/Core_Hound_Pack - 1st listed ability
	Problem 2
		Current Behavior
	Do not give the DoT debuff that does 50 damage every 30 seconds, and that stacks up to 10 times
		Expected Behavior
	Core hounds should place a debuff on their target that does 50 damage every 30 seconds, and it can stack up to 10 times
		Steps to Reproduce Problem
1. Pull Core hound
2. Observe Missing Debuff
Proof
	http://wowwiki.wikia.com/wiki/Core_Hound_Pack - 2nd Listed Ability
		



Problem 3
Current Behavior
	Core hound packs do not respawn in proper time frame
		Expected Behavior
	Core hound packs should respawn 1 hour after death, until Magmadar is dead
		Steps to Reproduce Problem
1. Kill Core hound
2. Wait and observe no respawn for a few hours
Proof
	http://wow.gamepedia.com/Core_Hound_pack - Notes section
	Problem 4
		Current Behavior
	Core Hounds do not pull as a pack
		Expected Behavior
	Core Hounds do not pull as a pack
		Steps to Reproduce
1. Taunt 1 core hound
2. View as only 1 or 2 come, not the entire pack
Proof
	https://www.youtube.com/watch?v=DCKVG75yCGw – Beginning Pull

Lava Surger – 2 Problems
		Current Behaviors
	Lava surgers don’t charge person with lowest threat
		Expected Behavior
	Lava surgers are supposed to charge the member in the raid will the lowest threat to them
		Steps to Reproduce
1. Take a group of people into MC
2. Pull Charger
3. Build threat and observe he never charges person with least threat
Proof
	http://wowwiki.wikia.com/wiki/Lava_Surger - 2nd listed ability
	
	Problem 2
		Current Behaviors
	Lava Surgers don’t respawn in proper time frame
		Expected Behvaiors
	Lava Surgers should respawn on an 28 minute timer, until Garr is killed
		Steps to Reproduce
1. Kill mobs
2. Wait for respawn and not have one
		Proof
	http://wowwiki.wikia.com/wiki/Lava_Surger - Notes section

Ancient Core Hound
		Current Behavior
	Ancient Core hounds do not respawn on timer
		Expected Behavior
	Ancient core hounds should respawn on an 18 minute timer, until Magmadar is killed
		Steps to Reproduce
1. Kill Ancient Core Hound
2. Wait 18 minutes and view no respawn
		Proof
	http://wowwiki.wikia.com/wiki/Ancient_Core_Hound - Notes Section





Flame Walkers
		Current Behavior
	Can be mind-controlled
		Expected Behavior
	Should be immune to mind-control
		Steps to reproduce
1. Try to mind-control
2. Succeed
		Proof
	-----------------------------------
Baron Geddon
		Current Behavior
	Baron Geddon begins to cast Inferno, but when he finishes casting nothing happens
		Expected Behavior
	A PBAoE attack will happen once Inferno finishes, that increases in damage with each pulse. The pulses come about 1 second apart and last for 10 seconds. Damage starts at 500 damage per pulse and increases by 500 every other pulse (500, 500, 1000, 1000, 1500, etc.). Baron Geddon will not move while casting this and he will do 6-10 pulses.
		Steps to Reproduce
1. Begin encounter with Baron Geddon
2. Wait For Inferno cast, and no actual spell
		Proof
	http://wowwiki.wikia.com/wiki/Baron_Geddon - Attacks and Abilities

Golemagg the Incinerator
		Current Behavior
	Golemagg can be pulled without his adds
		Expected Behavior
	Pulling Golemagg should pull the two core hounds near him
		Steps to Reproduce
1. Pull Golemagg
2. Observe as the 2 core hounds don’t come along with hin
		Proof
https://www.youtube.com/watch?v=u6hJR5Kw3ww – Beginning Pull

Sulfuron Harbinger
		Current Behavior
	Sulfuron’s Adds do not pull with him, they are all not linked
		Expected Behavior
	 They should be linked and all pull together
		Steps to Reproduce
1. Pull Mobs or boss
		Proof
	http://wowwiki.wikia.com/wiki/Sulfuron_Harbinger
Majordomo Executus
		Current Behavior
	His adds do not pull with him, they are all not linked together
		Expected Behavior
	They should be linked together and all pull together
		Steps to Reproduce
1. Pull one mob and observe how most of the group stays behind
		Proof	
	http://wowwiki.wikia.com/wiki/Majordomo_Executus






Ragnaros the Firelord
		Current behavior
	Ragnaros respawns every hour
		Expected Behavior
	Should not respawn
		Steps to Reproduce
1. Kill Rag
2. Wait an hour for respawn


Sorry bout this but i forgot 1 main bug! Garr Current Behavior Firesworn do not buff Garr on death Expected Behavior Firesworn should buff Garr on death, causing a crowd control Steps to Reproduce 1. Kill adds 2. View as buff does not get placed on Garr

**Comment by chaostheory102** [comment]
Aright! I shall be posting the Zul Gurub Bug List by tonight. Just gonna do
some final checks with guild 1st.

**Comment by chaostheory102** [comment]
Thank you for all the help!

**Comment by Rushor** [comment]
alright:

	Problem 1
		Current Behavior
Core Hounds in Magmadar’s Cavern do not resurrect one another when they are not killed within 10 second intervals
Expected Behavior
Core Hounds should revive one another if they 10 seconds have passed after one of them have died and 1 other member of the pack is still alive
Steps to Reproduce Problem
	1.Kill one of the core hound in the pack
	2. Wait 10 seconds
	3. Observe that it is not being respawned
		Proof
	http://wowwiki.wikia.com/wiki/Core_Hound_Pack - 1st listed ability

This cannot be fixed currently, because the core support for individual guid scripts is very poor.

**Comment by chaostheory102** [comment]
Okay, thank you for the update.

**Comment by Rushor** [comment]
@chaostheory102 

i could also implement and fix the respawnbehaviour for the corehounds now in https://github.com/dalaranwow/dalaran-wow/issues/4144 :)

Once the group was pulled, there will start a 10 second timer. Every 10 seconds all member will respawn if at least 1 member is still alive.

So all reported MC issues are fixed now, i will continue with ZG after i rechecked the timer for the abilities of all bosses :)

**Comment by chaostheory102** [comment]
Thank you so much!!

**Comment by chaostheory102** [comment]
are the fixes live or will they be implemented in next update

**Comment by Rushor** [comment]
The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by chaostheory102** [comment]
Ooh okay. Not to push our luck with this, but when we were first contacted
about bug lists and all that, we were told that we would be allowed to test
the changes before they were implemented. Is there any news on that?

**Comment by Rushor** [comment]
oh if this was said, then ofc you are allowed too to test this :)

**Comment by chaostheory102** [comment]
How do we go about that? is it the other server besides algalon?

**Comment by Rushor** [comment]
yes you will get in formations about this when the time is right. It's the PTR server where we test our changes :)

**Comment by chaostheory102** [comment]
oh okay! thank you then. Ill let our guild master know to just wait for
info!

**Comment by chaostheory102** [comment]
just to be aware, should we be waiting for an ingame message from a GM for
when the PTR is ready or an email?

**Comment by Rushor** [comment]
i will ask this in the staffmember chat how it will be handled - i will post it here once i got the answer :)

**Comment by chaostheory102** [comment]
sweet! thank you for all the help mate!

**Comment by Rushor** [comment]
@chaostheory102: A gamemaster will inform you ingame, one the changes can be tested by your guild. Cheers.

**Comment by chaostheory102** [comment]
oh nice! on Agdayne?

**Comment by Rushor** [comment]
on any member that will be online then

**Comment by chaostheory102** [comment]
okay! thank you for all the help

**Comment by chaostheory102** [comment]
So we did ZG and BWL today. You were correct about Broodlord Mandokir. He
should not be LOSd. As well he is poorly scripted, in that he does not 1
shot you after the glance(gaze/focus), but still charges you if you stand
still.
BWL list should be up within the next 48 hours.
And just to let  you know...there are some problems with INstant 70. A
guildmate of ours did it and was granted the lvl 70 gear but not actually
leveled up..

**Comment by Rushor** [comment]
alright, i think i can finish zg today.

> And just to let you know...there are some problems with INstant 70. A guildmate of ours did it and was granted the lvl 70 gear but not actually leveled up..

Please contant a gamemaster for this issue ingame. He will fix that for your mate :) 

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)

**Comment by chaostheory102** [comment]
Sweet! ill try to have BWL up as fast as i can!
Yeah thats what i told him! but better safe than sorry!

**Comment by Klala** [comment]
I am sorry to resurrect this issue but there is a problem with reputation gains from all the mobs in MC including bosses.
I gain 100 reputation from normal mobs and 500 from bosses (x5)
The reputation gains should be as seen in [here](http://wowwiki.wikia.com/wiki/Hydraxian_Waterlords?oldid=2366322)

Also, as I was on my way to magmadar, Ragnaros started yelling his summoning emotes, I was taking all the fire imps while I was doing so, I am investigating this matter right now.

**Comment by Rushor** [comment]
> When later I killed all the bosses and summoned him again the normal way, he was summoned correctly, meaning you can kill him twice.
> Also, as I was on my way to magmadar, Ragnaros started yelling his summoning emotes, I was taking all the fire imps while I was walking there, and indeed when I reached ragnaros' to see if he was indeed summoned for no reason, he was there standing.

There already exists a ticket for this. It's fixed on PTR.

> I am sorry to resurrect this issue but there is a problem with reputation gains from all the mobs in MC including bosses.

Could you open up a new issue for this and include all boss ids and include the reputation they should reward. Thank you!

**Comment by Klala** [comment]
Majordomo Executus along with his adds are also spawning at the wrong angle. A group of 60s could pull him and his adds before even reaching the top end/platform.
He should be positioned like shown in this [video](https://www.youtube.com/watch?v=vye62FQ26-s)
